science/projects/Ghazaleh Mahmoudi/01-main-content/main_content_detection.ipynb

```diff
@@ -0,0 +1,1400 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.        title = np.array(title.split())
</pre><p>As we talked in session, it is better to calculate split title once for each page and just pass it to function. In other words avoid computing it in all function calls.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>